install/kubernetes/cilium/templates/cilium-resource-quota.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if and (ne .Release.Namespace "kube-system" ) .Values.gke.enabled }}
```

Comment:
Don't you think it would be more useful if this was `if and (ne .Release.Namespace "kube-system" )`? ..or perhaps `if and (ne .Release.Namespace "kube-system" ) (not .resourceQuotas.enabled)`?

Reply:
E.g. it could be handy to add include this for OpenShift, where `cilium` namespace is also used.